### ISSUE TITLE
[BUG] Placeholder variables are empty in V2

### ISSUE THREAD

**Issue by Dada40** [SoMuchForSubtlety/f1viewer#145]
**Describe the bug**
Some or all placeholder variables (mentioned in the README) are not replaced with version 2 and result in an empty value.
The same config was working with V1.

**To Reproduce**
1. Config: Define a section in "custom_playback_options" that is using placeholder variables.
Example: "2021 - $event - $session - $perspective.mp4"
2. Start F1Viewer and start the custom playback option. Tested with 2021 Emilia Romagna GP Qualifying World Feed.
3. Resulting file name (in file system) doesn't have accurate values for placeholders.
Example: "2021 -  -  - .mp4"

**Expected behaviour**
Placeholder variables should be replaced with actual values as in V1.

**Desktop (please complete the following information):**
 - macOS Catalina 10.15.7
 - Installation method: Using raw binaries of "f1viewer_2.0.1_macOS_amd64.tar.gz"
 - Version:     2.0.1
Git commit:  f72b2875b558bd696655b6619bb2510e49f1e37a
Built:       2021-05-01T20:03:48Z

**Comment by charlesomer** [comment]
Not sure if this is related or not but it could be you are trying to connect in a country where F1TV Pro is not supported?

This build adds an error message for when this happens.
https://github.com/SoMuchForSubtlety/f1viewer/actions/runs/802890651

**Comment by Dada40** [comment]
> Not sure if this is related or not but it could be you are trying to connect in a country where F1TV Pro is not supported?

Actually yes (Germany). 
However I still have a valid and working F1TV Pro subscription, as my existing yearly subscription got renewed (which was the only possible way in Germany to keep F1TV Pro without any workarounds).

Streams with F1Viewer are still working great with my account, it's just the metadata (placeholder variables) that is missing. Can't tell whether this might be caused by some geoblocking. 
Maybe someone can test with an account / internet connection not located in Germany?

**Comment by charlesomer** [comment]
Yep I've just tried too and it didn't work so looks like it's definitely caused by the V2 api change, hopefully it's not too difficult to fix!

**Comment by SoMuchForSubtlety** [comment]
Yea, I still need to fill out metadata for v2 streams. This will be fixed in the next release.
